Documentation/markdown/IFC4.3XListOfChanges.md

```diff
@@ -0,0 +1,33 @@
+# List of changes between IFC4X3_RC4 and IFC4X3 versions
+
+### Missing data in OIP
+
+| Name                                 | IFC4X3_RC4.exp          | IFC4X3.exp            |
+|--------------------------------------|-------------------------|----------------------|
+| IfcCosine                            | `:heavy_check_mark:`    | `:x:`                |
+| IfcSine                              | `:heavy_check_mark:`    | `:x:`                |
+| IfcCosineSpiral                      |  `:x:`                  | `:heavy_check_mark:` |
+| IfcSineSpiral                        |  `:x:`                  | `:heavy_check_mark:` |
+
+
+### Entity definitions
+
+| Name                                         | IFC4X3_RC4.exp                                      | IFC4X3.exp                                                        |
+|----------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| IfcIndexedPolygonalFace inverses             | ['ToFaceSet']                                       | ['ToFaceSet', 'HasTexCoords']                                    |
+| IfcOpenCrossProfileDef attributes            | ['HorizontalWidths', 'Widths', 'Slopes', 'Tags']    | ['HorizontalWidths', 'Widths', 'Slopes', 'Tags', 'OffsetPoint']  |
+| IfcPolygonalFaceSet.Faces                    | Faces : list[1:?] of IfcIndexedPolygonalFace        | Faces : list[1:?] of unique IfcIndexedPolygonalFace              |
+| IfcSectionedSolidHorizontal attributes       | ['CrossSectionPositions', 'FixedAxisVertical']      | ['CrossSectionPositions']                                        |
+| IfcSweptDiskSolidPolygonal.FilletRadius      | FilletRadius : optional IfcPositiveLengthMeasure    | FilletRadius : optional IfcNonNegativeLengthMeasure              |
+
```

Comment:
Take a look at IfcObjectPlacement and IfcSectionedSurface. They might be relevant.

Reply:
I added IfcObjectPlacement, but not the IfcSectionedSurface, because I did not find any function that uses this entity.

Reply:
fine